productService/modules/getProductsList.ts

```diff
@@ -1,29 +1,37 @@
 import { APIGatewayProxyHandler, APIGatewayProxyEvent, Context } from 'aws-lambda';
-import products from '../mocks/products.json';
+import connectToDb from '../helpers/connectToDb';
 
 export const getProductsList: APIGatewayProxyHandler = async (event: APIGatewayProxyEvent, context: Context) => {
-  /* GET CAT PICTURE WITH AWAIT
-
-    import axios from 'axios'
-    const axios = require('axios');
-    const { data: { file } } = await axios.get('https://aws.random.cat/meow?ref=apilist.fun');
+  try {
+    console.log('method getProductsList was called without params');
```

Comment:
You should log event object even if it's not used in function code. Because it contains very useful information about request